src/websocket-client.ts

```diff
@@ -190,7 +190,7 @@ export class WebsocketClient extends EventEmitter {
     this.readyState = READY_STATE_CONNECTED;
 
     this._subscribe([...this._subscriptions]);
-    this.pingInterval = setInterval(this._ping.bind(this), this.options.pingInterval);
+    this.pingInterval = <any>setInterval(this._ping.bind(this), this.options.pingInterval);
```

Comment:
was this causing problems? Shouldn't need to cast to <any> ideally

Reply:
it was causing errors so 5 letters fixed